### ISSUE TITLE
Cannot compile

### ISSUE THREAD

**Issue by raffaelemancuso** [OtaK/hide-my-mess-rs#2]
```
$ cargo build --release
    Updating git repository `https://github.com/otak/nokhwa`
error: failed to resolve patches for `https://github.com/rust-lang/crates.io-index`

Caused by:
  failed to load source for dependency `nokhwa`

Caused by:
  Unable to update https://github.com/otak/nokhwa?branch=otak/buff-buffers#54d00bf3

Caused by:
  object not found - no match for id (54d00bf353c053a5a8a59f9371eaa1bbd46ee10f); class=Odb (9); code=NotFound (-3)
```

**Comment by OtaK** [comment]
Oh. That's because my PR went through on nokhwa. I'll fix it :)

**Comment by OtaK** [comment]
Fixed with 869ee012b36bf165cd6acc362c8e5b706d755a1f
